backend/scripts/start-on-testnet.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Stop and remove containers, networks
+echo "Stopping and removing containers, networks..."
+docker compose -f docker-compose-testnet.yaml --profile full down
+
+# Remove specified volumes to remove all state and start fresh
+echo "Removing specified volumes..."
+docker volume rm backend_redis_data
+docker volume rm backend_chainstorage
+docker volume rm backend_ipfs_data
```

Comment:
I think we might want to ask the user if they want to remove the ipfs data. If this is gone, it is gone and we have lost data on testnet (common, but I don't think that would be the expected outcome from a restart)